R/readme.R

```diff
@@ -11,7 +11,13 @@
 #' added to \code{.Rbuildignore}. The resulting README is populated with default
 #' YAML frontmatter and R fenced code blocks (\code{md}) or chunks (\code{Rmd}).
 #'
+#' \code{use_data_list_in_readme_rmd} and \code{use_data_list_in_readme_md}
+#' append a table to the README listing the datasets contained in the package.
 #' @inheritParams use_template
+#' @param stop_if_no_data Logical. If \code{TRUE},
```

Comment:
This was to address your previous concern about wanting to add datasets after the README has been created. It mostly makes sense if the user is creating a `README.md` directly (rather than `Rmd`), and they want to include datasets after they create this file.  I'll have a think about improving the documentation to clarify this.